docs/src/main/sphinx/release/release-427.md

```diff
@@ -0,0 +1,110 @@
+# Release 427 (25 Sep 2023)
+
+## General
+
+* Add support for IPv4 addresses in comparisons using the `contains` functions. ({issue}`18497`)
```

Comment:
IPv4 is already supported. This change adds support for comparing IPv4 addresses against IPv6 CIDRs and vice-versa.